packages/java-packages/snippet-replacer-maven-plugin/src/main/java/azuresdk/plugin/SnippetDictionary.java

```diff
@@ -0,0 +1,38 @@
+package azuresdk.plugin;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SnippetDictionary {
+    private HashMap<String, List<String>> _snippetDictionary = new HashMap<String, List<String>>();
```

Comment:
nit: don't use leading underscore for variable names.

Reply:
👍 next commit will change em.